### ISSUE TITLE
Is this plugin using apple IDFA? It shows up in a grep search.

### ISSUE THREAD

**Issue by epetre** [dpa99c/cordova-plugin-firebasex#569]
The documentation doesn't mention if/how IDFA is used, and apple is starting to enforce restrictions on December 8th. I can't find mentions on IDFA in the documentation and explanations, but I can find references to the IDFA in the code.

It shows up if we run ` grep -rl ./' -e 'advertisingIdentifier`
And we are concerned given that the deadline is approaching.

```
grep -rl './' -e 'advertisingIdentifier'
.//Pods/GoogleTagManager/Frameworks/GoogleTagManager.framework/GoogleTagManager
```

I was wondering if it was possible to either clarify or add it to the documentation or both?

Cheers and thank you for maintaining this plugin!!

**Comment by dpa99c** [comment]
The Google Tag Manager library (included by this plugin) is able to make use of IDFA but only if you explicitly enable it:

[Adding the Google Tag Manager SDK to Your Project](https://developers.google.com/tag-manager/ios/v3#add-sdk)

> If you would like your application to access the identifier for advertisers (IDFA) and tracking flag provided by that framework through the Google Tag Manager SDK macros, you'll also need to link these additional libraries:
>
> * `libAdIdAccess.a`
> * `AdSupport.framework`

This plugin includes neither of those so in order to use IDFA, you'd have to manually add them to your Xcode project.

TL;DR: by default this plugin doesn't use IDFA
